test/trento_web/controllers/v1/profile_controller_test.exs

```diff
@@ -25,6 +25,28 @@ defmodule TrentoWeb.V1.ProfileControllerTest do
      conn: put_req_header(conn, "accept", "application/json"), api_spec: api_spec, user: user}
   end
 
+  test "should disable write profile action when external idp integration is enabled", %{
```

Comment:
Let's put `idp` in uppercase as in other test descriptions

Reply:
done